src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+      )}
+      <p className="Person__partner">
+        {person.isMarried
+          ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
```suggestion
          ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```
person.sex === 'm' ? 'wife' : 'husband' can be moved to a const and reused here